### DISCUSSION TITLE
Gerador de nomes para cidades

### DISCUSSION THREAD

**Discussion by Nimerigar** [YuriAlessandro/gerador-ficha-tormenta20#340] [Ideias]
Oi pessoal, acho que seria bom um gerador de nomes para vilas e cidades, as vezes nós Mestres temos problemas em dar nomes para locais, que podem ser importantes nas nossas narrativas. Arton tem muitas comunidades e nem todas aparecem nos mapas oficiais.

**Comment by YuriAlessandro** [comment]
A ideia é ser um gerador totalmente aleatório ou só com nomes reais das cidades e vilas de Tormenta que existem nos mais diversos materiais?

Se for o primeiro caso, o "Fantasy Name Generator" já faz isso muito bem e muito além do que a gente poderia alcançar aqui nesse projeto pequeno. Dá uma olhada:

![image](https://user-images.githubusercontent.com/14967048/182636437-cd272dee-74df-4aad-b5f6-00db1f169d32.png)

Exemplo (nome de cidades de fantasia): https://www.fantasynamegenerators.com/fantasy-town-names.php

Se souber um pouco de inglês ou jogar no google (algumas vão traduzir, outras não) dá até para adaptar para ficar mais "abrasileirado" para o Tormenta (por exemplo "Whitepoint" pode virar "Pontobranco"). 

Agora se estamos falando de gerar nomes que já foram citados em materiais de tormenta, até da pra fazer. A questão é listar todos esses nomes em algum lugar para que possa ser utilizado no código (uma trabalheira gigante, eu imagino).
